docs/src/examples/components/Slider/Playground.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react'
+import { Slider } from '@stardust-ui/react'
+import { useBooleanKnob, useStringKnob } from '@stardust-ui/docs-components'
+
+const SliderPlayground: React.FunctionComponent = () => {
+  const [min] = useStringKnob({ name: 'min', initialValue: '0' })
+  const [max] = useStringKnob({ name: 'max', initialValue: '50' })
+  const [step] = useStringKnob({ name: 'step', initialValue: '1' })
+  const [value, setValue] = useStringKnob({ name: 'value', initialValue: '10' })
+
+  const [disabled] = useBooleanKnob({ name: 'disabled' })
+  const [fluid] = useBooleanKnob({ name: 'fluid' })
+  const [vertical] = useBooleanKnob({ name: 'vertical' })
+
+  return (
+    <Slider
+      disabled={disabled}
+      fluid={fluid}
+      min={min}
+      max={max}
+      step={step}
+      value={value}
+      vertical={vertical}
+      onChange={(e, data) => setValue(String(data.value))}
```

Comment:
https://github.com/stardust-ui/react/pull/1559/files#r302911394
Do we need this type cast?

Reply:
yes; reasons:
- I don't think https://github.com/stardust-ui/react/pull/1559/files#r302911394 is the correct fix; the value can be number as well
- `useStringKnob` requires `string` and `value` coming from `Slider` has type `string | number`

Reply:
```
      const value = _.get(e, 'target.value')
      _.invoke(this.props, 'onChange', e, { ...this.props, value })
```

According to these lines it can't be `string | number`:

```js
const e:  React.ChangeEvent<HTMLInputElement> = {}
e.target.value // is `string`
```

See https://github.com/stardust-ui/react/pull/1559#discussion_r302911394

Reply:
well that was a good argument 😄 applied suggestion, we should find a way to generalize this. thanks, resolving